src/app/lecture/knowledge-node/notes/notes.component.html

```diff
@@ -0,0 +1,26 @@
+<div id="notes" fxLayout="column">
+  <div *ngIf="edit" class="mat-card" fxLayout="column">
+    <ngx-mat-markdown-text-editor
+      [(ngModel)]="text"
+      livePreviewEnabled="true"
```

Comment:
Is it possible to configure the live preview to be less noticable (e.g., have it be twice as small as the actual editor)?

Reply:
Not with the editor we are currently using, unfortunately.